holopy/inference/prior.py

```diff
@@ -211,7 +235,20 @@ def __neg__(self):
 
 class BoundedGaussian(Gaussian):
     # Note: this is not normalized
-    def __init__(self, mu, sd, lower_bound=-np.inf, upper_bound=np.inf, name=None):
+    def __init__(self, mu, sd, lower_bound=-np.inf, upper_bound=np.inf,
+                 name=None):
+        """
+        Gaussian prior restricted to an interval
```

Comment:
Might be worth mentioning here that the prob and lnprob values it returns are not correctly normalized - i.e. they should be treated as relative probabilities only.

Reply:
Should be fixed by 68438b8